src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdProviderMetadataNames.cs

```diff
@@ -51,6 +51,9 @@ public static class OpenIdProviderMetadataNames
         public const string IdTokenEncryptionAlgValuesSupported = "id_token_encryption_alg_values_supported";
         public const string IdTokenEncryptionEncValuesSupported = "id_token_encryption_enc_values_supported";
         public const string IdTokenSigningAlgValuesSupported = "id_token_signing_alg_values_supported";
+        public const string IntrospectionEndpoint = "introspection_endpoint";
```

Comment:
@kevinchalet is there a spec we can reference for these values?

Reply:
Yep, these values are defined in the OAuth 2.0 Authorization Server Metadata specification (RFC8414): https://tools.ietf.org/html/rfc8414#section-2

Reply:
@kevinchalet super thanks!